PWGCF/MultiparticleCorrelations/Core/MuPa-MemberFunctions.h

```diff
@@ -2826,35 +2856,31 @@ void BookParticleCutsHistograms()
   }
 
   // c) Book the particle cut counter (absolute):
-  TString srs[2] = {"rec", "sim"};
-  TString srs_long[2] = {"reconstructed", "simulated"};
-  TString scc[eCutCounter_N] = {"abs", "seq"};
-  TString scc_long[eCutCounter_N] = {"absolute", "sequential"};
-
+  // ...
   for (Int_t rs = 0; rs < 2; rs++) // reco/sim
   {
 
     if (Skip(rs)) {
       continue;
     }
 
-    for (Int_t cc = 0; cc < eCutCounter_N; cc++) // enum eCutCounter
+    for (Int_t cuco = 0; cuco < eCutCounter_N; cuco++) // cut counter. I have to use "cuco", because "cc." is already reserved for struct "common cosmetics"
     {
```

Comment:
> I have to use "cuco", because "cc." is already reserved

Don't limit yourself and be more readable! Suggestion: `ixCutCounter`

Reply:
I use this one as an array index, so its name preferably has to stay short. But I will re-think the whole thing